.gitlab-ci.yml

```diff
@@ -5,7 +5,7 @@ stages:
   - trigger
 
 variables:
-  GOLANG_VERSION: "1.17.6"
+  GOLANG_VERSION: "1.17.13"
```

Comment:
can we take this opportunity to verify what we can bump the version of go in the acceptance tests? last time I checked it was 1.12 what version does kirkstone have?

Reply:
> can we take this opportunity to verify what we can bump the version of go in the acceptance tests?

We cannot bump the version there, we are at the mercy of whatever Yocto uses. If it was 1.12, it probably still is. Any vulnerabilities will have to be handled by the Yocto people (but we can report it, of course).